### ISSUE TITLE
TeslaUsb not detected by Windows or Model 3

### ISSUE THREAD

**Issue by firedfly** [cimryan/teslausb#19]
I completed the configuration instructions up to the last part about _Get the Pi set up for your Tesla_.  Windows would never see the Pi as a USB drive.  I manually created the _TeslaCam_ directory on the Pi via ssh and tried to connect to my car.  The Model 3 never detected the Pi either.

After much debugging, I discovered the issue was with the USB cable I was using.  Once I swapped cables, the Pi was detected by Windows as a USB drive and the Tesla started to use it.

TLDR: check your USB cable if your Pi is not detected after configuration

**Comment by firedfly** [comment]
Closing as I already solved this issue.
